[Github] Updates to readme and scope (#121)
AuroraLegacy/elements: .github/CONTRIBUTING.md

<a name="pull-requests"></a>
## Pull Requests

> Good pull requests are a great help. They should remain focused in scope and avoid containing unrelated commits. Make sure to **not** create one from your master branch, instead create a new branch for it.  All pull requests will go through the [review process](#review-process).  Small, focused pull requests are typically quicker to review and merge, but it is better to combine pull requests when making multiple changes within the same file(s) or focus.

Reviewer: two more double spaces
```suggestion
Good pull requests are a great help. They should remain focused in scope and avoid containing unrelated commits. Make sure to **not** create one from your master branch, instead create a new branch for it. All pull requests will go through the [review process](#review-process). Small, focused pull requests are typically quicker to review and merge, but it is better to combine pull requests when making multiple changes within the same file(s) or focus.
```

* Name your pull request with the format `[Source Abbreviation] %Specific Title%`. Use `[MISC]` when modifying files from multiple sources (Example: `[PHB] Fireball description fixes`)
* Give your commits a meaningful name instead of just _updated file1.xml_
* When making changes to an existing file, increase the version number by 1 (Example: from 0.1.2 to 0.1.3), no matter number of the commits.
* Indent the files properly (using tabs, size 4) to make them readable
* Make sure the IDs of new elements only contain alphanumeric characters in upper case, underscores (`[A-Z]`, `[0-9]`, `_`) and no spaces or other special characters.
* Make sure to check and test your files - See [review process](#review-process)
* Ensure the contribution is complete and ready to be checked and merged before opening a pull request. If you're working on something large, you can create an issue with a todo list and/or open a draft pull request to communicate your progress.

<br>

<a name="review-process"></a>
## Review Process

Anyone is welcome to assist in the review process and it is a huge help.

> All pull requests require 2 or more reviews depending on the complexity and the number of files/lines being changed. It is uncommon that a file will be perfect when starting the review process.  Approving a pull request with problems or requesting changes without appropriate context may cause a review to be ignored as part of the review process.

Reviewer: double space
```suggestion
All pull requests require 2 or more reviews depending on the complexity and the number of files/lines being changed. It is uncommon that a file will be perfect when starting the review process. Approving a pull request with problems or requesting changes without appropriate context may cause a review to be ignored as part of the review process.
```

A good review will be different depending on the element(s) being changed or added. Below is a basic list of things to check for:

#### Overall:
* Check that the file does not contain errors on launch and appears in the app
* Check the element(s) and see that the description is as close to the source material as possible
* Check that sources/spells are in italics
* Read every description for common copy paste issues (Examples: `l` instead of `1`, random spaces in the middle of words, random spaces around punctuation)
* Check the sheet for `==UNKNOWN_VALUE==` where a variable should be
* Check that unnecessary elements are hidden from the compendium (Example: helper elements for selecting spellcasting ability, duplicate growing items within the same source)
* Check that the version number increases 1 time per updated file
* Check that new files have correct links
* Check IDs for proper format and all upper case

#### Races / Subraces / Classes / Archetypes:

> * Load the element(s) at lv 1, lv 20, and the level before and after a feature is added or text is changed (it is best to just step through every level and see the sheet)

Reviewer: ```suggestion
* Load the element(s) at each level before and after a feature is added or text is changed (it is best to just step through every level and check the sheet preview)
```
